server/src/test/java/io/pravega/schemaregistry/storage/impl/group/GroupPravegaTest.java

```diff
@@ -0,0 +1,757 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.schemaregistry.storage.impl.group;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hashing;
+import io.pravega.client.ClientConfig;
+import io.pravega.schemaregistry.common.Either;
+import io.pravega.schemaregistry.contract.data.Compatibility;
+import io.pravega.schemaregistry.contract.data.EncodingId;
+import io.pravega.schemaregistry.contract.data.EncodingInfo;
+import io.pravega.schemaregistry.contract.data.GroupHistoryRecord;
+import io.pravega.schemaregistry.contract.data.GroupProperties;
+import io.pravega.schemaregistry.contract.data.SchemaInfo;
+import io.pravega.schemaregistry.contract.data.SchemaWithVersion;
+import io.pravega.schemaregistry.contract.data.SerializationFormat;
+import io.pravega.schemaregistry.contract.data.VersionInfo;
+import io.pravega.schemaregistry.pravegastandalone.PravegaStandaloneUtils;
+import io.pravega.schemaregistry.server.rest.ServiceConfig;
+import io.pravega.schemaregistry.service.Config;
+import io.pravega.schemaregistry.storage.Etag;
+import io.pravega.schemaregistry.storage.SchemaStore;
+import io.pravega.schemaregistry.storage.SchemaStoreFactory;
+import io.pravega.schemaregistry.storage.StoreExceptions;
+import io.pravega.schemaregistry.storage.client.TableStore;
+import io.pravega.schemaregistry.storage.client.Version;
+import io.pravega.schemaregistry.storage.impl.group.records.NamespaceAndGroup;
+import io.pravega.schemaregistry.storage.impl.group.records.TableKeySerializer;
+import io.pravega.schemaregistry.storage.impl.group.records.TableRecords;
+import io.pravega.schemaregistry.storage.impl.groups.GroupsValue;
+import io.pravega.schemaregistry.storage.impl.groups.PravegaKVGroups;
+import io.pravega.test.common.AssertExtensions;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static org.junit.Assert.*;
+
+public class GroupPravegaTest {
+    static final String GROUPS = "_schemaregistry/groups/0";
```

Comment:
dont define this here.. use it from the code class.. change the visibility of the constant in the class if required.